### PR TITLE
basic volunteer home page

### PR DESCRIPTION
**Pull request by pingchenchan**
## Description

basic volunteer home page without volunteer account switching and session timeout features

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

_Link user story from projects.digitalaidseattle.org_

- Related Issue # [PIT-122](https://das-ph-inventory-tracker.atlassian.net/jira/core/projects/PIT/board?selectedIssue=PIT-122)

## QA Instructions, Screenshots, Recordings
Screenshots
image

### CODE REVIEW

File: src/pages/VolunteerHome/index.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Box, Typography, Button, Grid } from '@mui/material';
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import AddIcon from '@mui/icons-material/Add';
+import { useNavigate } from 'react-router-dom';
+
+const VolunteerHome: React.FC = () => {
+  const navigate = useNavigate();
```
Comment (bospoort) [inline src/pages/VolunteerHome/index.tsx:8]: nit: add whitline

**Review by bospoort — COMMENTED** [review]
LGTM

**Review by aitzeng — APPROVED** [review]

**Comment by aitzeng** [comment]
Wasn't able to run this locally due to the accounts not working, but the code looks good.

**Comment by bospoort** [comment]
lgtm still

**Review by bospoort — APPROVED** [review]

**Pull request merged**